flight/goto/goto.py

```diff
@@ -0,0 +1,58 @@
+"""
+File containing the move_to function responsible
+for moving the drone to a certain waypoint
+"""
+import typing
+import asyncio
+from mavsdk import System
+import logging
+import math
+
+async def move_to(drone: System, latitude: float, longitude: float, altitude: float) -> None:
+    """
+    This function takes in a latitude, longitude and altitude and autonomously
+    moves the drone to that waypoint. This function will also auto convert the altitude
+    from feet to meters
+
+    Args:
+        drone (System): a drone object that has all offboard data needed for computation
+        latitude (float): a float containing the requested latittude to move to
+        longitude (float): a float containing the requested longitude to move to
+        altitude (float): a float contatining the requested altitude to go to (in meters)
+
+    Returns:
+        None
```

Comment:
We're using the numpy documentation standards for our code, so there are no content issues here, you just need to change the format. Check the slides for the precise formatting of headings & content.